dbcrossbarlib/src/drivers/bigquery_shared/column.rs

```diff
@@ -57,8 +58,10 @@ impl BqColumn {
             }
             BqDataType::NonArray(ty) => (ty, Mode::Required),
         };
+        let spaces_and_dashes_re = Regex::new(r"[-\s]").unwrap();
+        let new_column_name = spaces_and_dashes_re.replace_all(&col.name, "_");
```

Comment:
i think you have to escape the dash in the character class

e.g. `[\-\s]`

otherwise it's treated as a range

Reply:
I get a `Syntax(Error { pos: 2, surround: "[\\-\\s]", kind: UnrecognizedEscape('-') })'` if I escape the dash